src/modules/hashivault/config.rs

```diff
@@ -13,6 +13,26 @@ pub struct HashiVaultInput {
 }
 
 
+#[derive(Debug, Deserialize, Clone, PartialEq, JsonSchema)]
+#[serde(tag = "type")]
+pub enum HashiVaultAuth {
```

Comment:
Is it possible to have an `Option<String>` here with the "usual" path? eg. `auth/kubernetes` for Kubernetes, etc. 

Not blocking for merge, I can update it myself later :)

Reply:
Yes, I changed to use default values for the mount path.